pkg/topology/builder.go

```diff
@@ -44,16 +44,16 @@ func (b *Builder) Build(resourceFilter *mk8s.ResourceFilter) (*Topology, error)
 		b.evaluateService(res, topology, svc)
 	}
 
-	// Populate services with traffic-target definitions.
-	for _, tt := range res.TrafficTargets {
-		b.evaluateTrafficTarget(res, topology, tt)
-	}
-
 	// Populate services with traffic-split definitions.
 	for _, ts := range res.TrafficSplits {
 		b.evaluateTrafficSplit(topology, ts)
 	}
 
+	// Populate services with traffic-target definitions.
```

Comment:
Why was it moved? Is there a specific reason?

Reply:
It had to be moved since `evaluateTrafficTarget` needs to know if there's already a TrafficSplit on a service. So TrafficSplits have to be evaluated first.